### ISSUE TITLE
[FR] Make termgraph can plot histogram

### ISSUE THREAD

**Issue by HiromuIshikawa** [mkaz/termgraph#50]
I want to plot histogram by Termgraph.

When I want to visualize some data, I often want to see the distribution by histogram.
So, given the use case of termgraph, I think it is the function we want

Example of histogram by Pandas is as follow:
```
data = np.random.normal(10, 10, 1000) # generate dummy data follow normal distribution
df = pd.DataFrame({'col': data})
df.hist()
```
![normalhist](https://user-images.githubusercontent.com/15909860/69234360-495b9d80-0bd2-11ea-8ee9-17d1a387441a.png)

My expected output example in termgraph is as follow:
```
$ termgraph hist.dat --histogram --bins 5
-20 - -10: ▇▇▇▇▇▇▇ 7
-10 -   0: ▇▇▇▇▇▇▇▇▇ 9
  0 -  10: ▇▇▇▇▇▇▇▇▇▇▇ 11
 10 -  20: ▇▇▇▇▇▇▇▇▇ 9
 20 -  30: ▇▇▇▇▇▇▇ 7
```

If it is good, I will work on this function and make a PR.

**Comment by HiromuIshikawa** [comment]
My expected output example in termgraph is as follow:
```
$ termgraph hist.dat --histogram --bins 5
-20 - -10: ▇▇▇▇▇▇▇ 7
-10 -   0: ▇▇▇▇▇▇▇▇▇ 9
  0 -  10: ▇▇▇▇▇▇▇▇▇▇▇ 11
 10 -  20: ▇▇▇▇▇▇▇▇▇ 9
 20 -  30: ▇▇▇▇▇▇▇ 7
```

**Comment by mkaz** [comment]
Closed in #51

**Comment by gennaro-tedesco** [comment]
Is it possible to read up on the documentation for the right grammar to use `--histogram`? It seems to be missing from the README (it is unclear what data format must be fed into it).

I would expect something along the lines of 
```
seq 10 | termgraph --histogram
```
to work, however I get an exception
```
Traceback (most recent call last):
  File "/usr/local/bin/termgraph", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/termgraph/termgraph.py", line 138, in main
    chart(colors, data, args, labels)
  File "/usr/local/lib/python3.9/site-packages/termgraph/termgraph.py", line 539, in chart
    for row in hist_rows(data, args, colors):
  File "/usr/local/lib/python3.9/site-packages/termgraph/termgraph.py", line 220, in hist_rows
    val_min = find_min(data)
  File "/usr/local/lib/python3.9/site-packages/termgraph/termgraph.py", line 145, in find_min
    return min([min(sublist) for sublist in data])
  File "/usr/local/lib/python3.9/site-packages/termgraph/termgraph.py", line 145, in <listcomp>
    return min([min(sublist) for sublist in data])
ValueError: min() arg is an empty sequence
```
(and similar passing actual csv files).
